### ISSUE TITLE
Codepaths surrounded by quotes fail to work

### ISSUE THREAD

**Issue by Aidan63** [lix-pm/haxeshim#32]
Hello,
I've recently started using openfl's hxp library to generate hxml files. When you use hxp's HXML tool it will wrap any codepaths you specify in quotes, while this works fine with normal haxe with lix you get a 'classpath is not a directory or cannot be read from' error.
The culprit seems to be this line https://github.com/lix-pm/haxeshim/blob/master/src/haxeshim/HaxeCli.hx#L56 as I did some tests and haxe will throw an exception when pass a string surrounded in quotes to readDirectory. My solution for now was to modify hxp to not put quotes around the code paths but it would be nice to have haxeshim match haxe in this case.

Cheers.

**Comment by back2dos** [comment]
So stripping any leading and trailing single and double quotes would do the trick?

**Comment by kevinresol** [comment]
Maybe a good chance to factor out the hxml-parsing logic as a separate library?

**Comment by Aidan63** [comment]
I think removing leading and trailing quotes would work, stopping hxp from adding them was the only thing I need to change to get it working with lix.

**Comment by back2dos** [comment]
> Maybe a good chance to factor out the hxml-parsing logic as a separate library?

Yes, although my gut feeling is that that's easier said than done :D
